.github/workflows/black.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           black .
           git config --global user.name 'Geek.Zone/Bot'
           git config --global user.email 'geekzonebot@users.noreply.github.com'
-          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORYcheck
```

Comment:
I don't think we want this. also, I have fixed black in #588

Reply:
Good Spot, must've been a typo 😰